colcon_cmake/test_result/ctest.py

```diff
@@ -55,7 +55,7 @@ def get_test_results(
             if root.tag != 'Site':
                 logger.warning(
                     "Skipping '{latest_xml_path}': the root tag is not 'Site'"
-                    .format_map(locals()))
+                        .format_map(locals()))
```

Comment:
Please revert unrelated changes which by the way also fails the unit tests.

Reply:
thanks! Just realized that :-/